pytorch_lightning/trainer/trainer.py

```diff
@@ -2599,10 +2599,10 @@ def logger(self) -> Optional[LightningLoggerBase]:
         if len(self.loggers) == 1:
             return self.loggers[0]
         else:
-            rank_zero_warn(
-                "Using trainer.logger when Trainer is configured to use multiple loggers."
-                " This behavior will change in v1.8 when LoggerCollection is removed, and"
-                " trainer.logger will return the first logger in trainer.loggers"
+            rank_zero_deprecation(
```

Comment:
I think this should be `rank_zero_warn` because otherwise the framework will emit dep warnings every time it accesses `trainer.logger` when we have multiple loggers.